host/xscope_io_host.c

```diff
@@ -97,6 +116,16 @@ int send_file_chunk(unsigned file_idx, unsigned req_size)
     return(0);
 }
 
+/**
+ * @brief Record xscope data
+ * 
+ * @param id id of the data
+ * @param timestamp timestamp of the data
+ * @param length length of the data
+ * @param dataval data value
+ * @param databytes data bytes
```

Comment:
What's the difference between "data value" and "data bytes"?

Reply:
data value is single char is produced by xscope_char 
and data_bytes char[] by xscope_bytes 
xscope_record is generic to recieve any type, in this particular case we dont use data as we always transmit bytes. I guess it was easier like this for both device and host.

Reply:
changed to me a bit more explicit